WallPaperDev/WallPaperDev/Views/SearchImages Views/SearchImagesCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  SearchImagesCell.swift
+//  WallPaperDev
+//
+//  Created by Jamar Gibbs on 11/5/19.
+//  Copyright © 2019 Stephen Ouyang. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SearchImagesCell : UICollectionViewCell {
+  let identifier = "searchCell"
+  
+  // Using layers to hold uninteractable images
+  lazy var photoLayer = CALayer()
+  lazy var maskLayer = CAShapeLayer()
+  lazy var borderLayer = CAShapeLayer()
+  
```

Comment:
Are you noticing any flickering for the cells? May make sense to when a cell is preparing to be reused we empty out the old CALayer? Thoughts?

Reply:
Oh yeah I just see it now. I'll issue a bug to work on this later.